extensions/amp-pan-zoom/amp-pan-zoom.md

```diff
@@ -51,5 +51,17 @@ limitations under the License.
   </amp-layout>
 ```
 
-## Events
-It exposes the `zoomEnd` action, which contains the single value, `scale`. This value contains the current scale of the child content being zoomed.
\ No newline at end of file
+## Attributes
+##### max-scale (optional)
+This attribute determines the max allowed zoom scale of the content being zoomed. The default value for this is 3.
+
+##### initial-scale (optional)
+This attribute allows developers to specify a beginning zoom scale at initialization. The default initial scale is 1.
```

Comment:
Wordy, cut to "Specifies a default zoom scale. Otherwise set to 1."

And the other attributes too.

Reply:
Done.